notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -343,9 +440,73 @@
     "\n",
     "    rows.append(add)\n",
     "icrp = icrp.append(rows, ignore_index=True)\n",
-    "file_NDX.close()\n",
+    "file_NDX.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add stable progeny to dataframe, using mass data from the AME2020 database. Set their half life to infinity, and their number of decay modes to 0."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "stable_dict = {}\n",
+    "stable_df = pd.DataFrame()\n",
+    "# ignore empty data, SF (spontanious fission)\n",
+    "stable_name_list = [\"\",\"SF\"]\n",
+    "sf_key = 0\n",
+    "progeny_list = (list(icrp.Progeny_1)\n",
+    "                + list(icrp.Progeny_2)\n",
+    "                + list(icrp.Progeny_3)\n",
+    "                + list(icrp.Progeny_4))\n",
+    "\n",
+    "for n,i in enumerate(progeny_list):\n",
+    "    if (i not in list(icrp.Radionuclide) and\n",
+    "        i not in stable_name_list and str(i) != \"nan\"):\n",
+    "\n",
+    "        stable_progeny = []\n",
+    "        name = i\n",
+    "        \n",
+    "        if name == \"Ta-180m\": # only metastable stable nuclide\n",
+    "            # pull catagorical data from ame2020, isomer excitation energy from nubase\n",
+    "            ame_info = ame2020_df.loc[\"Ta-180\"]\n",
+    "            stable_progeny  = [i, i.split(\"-\")[0], int(ame_info[\"Z\"]),\n",
+    "                               int(ame_info[\"A\"]), \"\", ame_info[\"Mass\"]\n",
+    "                               + nubase_df.loc[\"Ta-180m\"].Isomer_excitation_energy]\n",
```

Comment:
Great spot finding this stable isomer! Can you also add the char `m` into the Metastable_state column (second argument on line 482)?